README.md

```diff
@@ -920,6 +938,251 @@ branch. Right? **Right?**
 
 So you need to merge your branch into `main`.
 
+We don't, however, want to just smash our stuff into `main`. We want to
+take advantage of our very clever colleagues and have at least one other
+person perform a code review on our proposed changes. A code review is
+essentially an online conversation where members of the team build a
+shared understanding of these changes, proposing fixes and improvements,
+and generally working together to make the code base as good as you
+collectively can.
+
+#### Create a pull request
+
+Go this repo on GitHub (on the web) and there's likely to be one or
+more messages in a light yellow box near the top that say something like:
+
+> **\<branch name\>** had recent pushes \<some recent time frame\>
+
+If you pushed recently, then one of these messages may will be about your
+branch.
+
+* If there is such a message for your branch, click "Compare & pull request".
+* If there isn't a message for your branch:
+  * Select your branch from the branch dropdown. Then there should be a message
+    for your branch.
+  * Click "Compare & pull request".
+
+That will bring up the page for creating pull requests, filling in some of the
+information based on your commits on that branch. You want to fill in the title
+and body of the pull request in a way that describes the goals and actions of
+this pull request – what are you trying to accomplish and how did you do it?
+
+When all that's ready, click the green "Create pull request" button to actually
+create the pull request.
+
+Once that's done, you want to request two or three reviewers. Click the gear
+icon to the right of "Reviewers" in the right hand column of actions. Hopefully
+there will be several people in the drop-down list; if so pick two or three at
+random. You should always be able to just type in "NicMcPhee" to add me as a
+reviewer, but you want to get a few other people in case I'm swamped.
+
+#### You can't merge yet
+
+What you _want_ to do is merge this into `main` so your wonderful changes are
+an official part of the project. You can't do it yet, though – the "Merge pull
+request" button down at the bottom is grayed out. Before you can merge, at least
+two things must be true:
+
+* A set of automated checks have to pass
+* Your pull request/branch has to be up-to-date with `main`
+* At least one person has to review _and accept_ your pull request
```

Comment:
Maybe we want to require 2 reviews for this lab? That'll make it more likely everyone will end up _giving_ a code review.

Reply:
Excellent point. I'll change that. We probably don't want to change it in the UMM-CSci-3601 org since we often won't have two people to provide reviews, so I'll add that to the things a faculty member needs to change when they copy it into the semester-specific org.